src/plugins/unified_search/public/query_string_input/query_bar_menu.tsx

```diff
@@ -267,7 +267,6 @@ function QueryBarMenuComponent({
       anchorPosition="downLeft"
       repositionOnScroll
       data-test-subj="queryBarMenuPopover"
-      hasDragDrop
```

Comment:
I'm not totally sure why the popovers in this component had the `hasDragDrop` prop in the first place, as they don't seem to contain any draggable content. @davismcphee any idea? (pinging you as the last git history that touched this line)

Reply:
Hey Cee, the draggable content is in the filter editor when "Add filter" is clicked:
![image](https://github.com/user-attachments/assets/8e7fc18b-7db5-46bf-bd51-7f98424bba1e)

Reply:
Ahhhh, thank you so much! I've pushed up a replacement of the prop in https://github.com/elastic/kibana/pull/194619/commits/9d25b800709c3f2e35d40e9c91bcfe5377082814